### ISSUE TITLE
GriddedReferenceGraphicTools toolbox does not open in ArcGIS Pro 2.0

### ISSUE THREAD

**Issue by elinz** [Esri/solutions-geoprocessing-toolbox#611]
The GRG toolbox is being supported on ArcGIS Pro 2.0. When add the GRG toolbox to Pro 2.0 the tool does not open:
![image](https://user-images.githubusercontent.com/4635490/30833739-29fa3246-a205-11e7-8dbf-a86c27f12dcf.png)

Tested with the gp tools in "mf-581-mgrs-in-grg" as of 25 Sept 2017 15:20.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by Dbarnes1** [comment]
@mfunk I tested this in the latest 2.1 as well and its not working either

**Comment by Dbarnes1** [comment]
Took a screenshot of the python traceback errors in this review: https://github.com/Esri/solutions-geoprocessing-toolbox/pull/608#pullrequestreview-65060720

**Comment by mfunk** [comment]
```
Parsing error: 
Traceback (most recent call last):
  File "D:\GitHub\solutions-geoprocessing-toolbox\clearing_operations\scripts\GRGTools.py", line 39, in <module>
    from . import RefGrid
  File "D:\GitHub\solutions-geoprocessing-toolbox\clearing_operations\scripts\RefGrid.py", line 9, in <module>
    import Utilities
ImportError: No module named 'Utilities'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 31, in <module>
  File "D:\GitHub\solutions-geoprocessing-toolbox\clearing_operations\scripts\__init__.py", line 2, in <module>
    from . import GRGTools
  File "D:\GitHub\solutions-geoprocessing-toolbox\clearing_operations\scripts\GRGTools.py", line 41, in <module>
    import RefGrid
ImportError: No module named 'RefGrid'
```

@elinz @Dbarnes1 Pushed changes that seem to fix this for Pro 1.4. Let me know if it works or not. If not please right-click on the busted toolbox and select **Check Syntax**. Paste the errors in the dialog box into new comment here. Thanks.

**Comment by Dbarnes1** [comment]
@mfunk this worked, I'm going to go ahead and close
